docs/quickstart/multisig-utxos-with-avalanchejs.md

```diff
@@ -0,0 +1,1145 @@
+
+![](/img/multisig-utxos-1.png)
+
+
+## Multi-Signature UTXOs
+
+UTXOs can be associated with multiple addresses. If there are multiple owners of a UTXO, then we must note the `threshold` value. We have to include signatures of a threshold number of UTXO owners with the unsigned transaction to consume UTXOs present in the inputs. The threshold value of a UTXO is set while issuing the transaction.
+
+We can use these multi-sig UTXOs as inputs for multiple purposes and not only for sending assets. For example, we can use them to create subnets, add delegators, add validators, etc.
+
+## Atomic Transactions
+
+On Avalanche, we can even create cross-chain outputs. This means that we can do a native cross-chain transfer of assets. These are made possible through **Atomic Transactions**. This is a 2-step process -
+* Export transaction on source chain
+* Import transactions on the destination chain
+
+Atomic transactions are similar to other transactions. We use UTXOs of the source chain as inputs and create outputs owned by destination chain addresses. When the export transactions are issued, the newly created UTXOs lie in the **Exported Atomic Memory**. These are neither on the source chain nor on the destination chain. These UTXOs can only be used as inputs by their owners on the destination chain while making import transactions. Using these UTXOs on the atomic memory, we can create multiple outputs with different amounts or addresses.
+
+![](/img/multisig-utxos-2.png)
+
+## UTXOs on C-Chain
+
+We can't use UTXOs on C-Chain to do regular transactions because it follows the account-based approach. In C-Chain, each address (account) is mapped with its balance, and the assets are transferred simply by adding and subtracting from this balance using the virtual machine.
+
+But we can export UTXOs with one or multiple owners to C-Chain and then import them by signing the transaction with the qualified spenders containing those UTXOs as inputs. The output on C-Chain can only have a single owner (a hexadecimal address). Similarly while exporting from C-Chain to other chains, we can have multiple owners for the output, but input will be signed only by the account whose balance is getting used.
+
+## Getting Hands-On Multi-Signature UTXOs
+
+Next, we will make utility and other helpful functions, so that, we can use them to create multi-sig UTXOs and spend them with ease. These functions will extract common steps into a function so that we do not have to follow each step every time we are issuing a transaction.
+
+**You can either follow the steps below to get a better understanding of concepts and code or directly clone and test the examples from this [repo](https://github.com/rajranjan0608/multisignature-utxos).**
```

Comment:
You should create a new project in Ava Labs' github, or add this as examples in AvalancheJS. We should avoid reference to personal github when possible.